session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1453,6 +1453,10 @@ private int calculateLength(List<TSDataType> types, List<Object> values)
   private void putValues(List<TSDataType> types, List<Object> values, ByteBuffer buffer)
       throws IoTDBConnectionException {
     for (int i = 0; i < values.size(); i++) {
+      if (values.get(i) == null) {
+        ReadWriteIOUtils.write((byte) -2, buffer);
```

Comment:
Maybe this "-2" can be a constant with a understandable name.

Reply:
Sure